backend/src/database/models/users.ts

```diff
@@ -0,0 +1,101 @@
+const { MongoClient } = require("mongodb");
+
+async function main() {
+  const uri = `mongodb+srv://noahmilliken:creamfilling@oreocluster.kasbgor.mongodb.net/?retryWrites=true&w=majority`;
```

Comment:
We need to move this to a .env file and import it from there or everyone who sees our repo will have access to our database

Reply:
In terms of import syntax are we using es6 or common js. I have seen both. does it matter?

Reply:
es6, lets keep it consistent